Makefile

```diff
@@ -387,32 +398,66 @@ __check_defined = \
     $(if $(value $1),, \
     $(error Undefined make flag: $1$(if $2, ($2))))
 
-# Flash the compiled
-flash: $(BUILD)/$(OUT_FILE)-nosd.hex
+	
+#------------------- Flash with NRFUTIL -------------------
+ifeq ($(FLASHER),nrf)

```

Comment:
nrfjprog is more appropriate since we are referring to the flash tool/command. nrf is too generic.